docs/source/1.0/spec/aws/aws-ec2-query-protocol.rst

```diff
@@ -378,15 +351,16 @@ serialized in the response.
 
 .. code-block:: xml
 
-    <Errors>
-        <Error>
-            <Type>Sender</Type>
-            <Code>InvalidGreeting</Code>
-            <Message>Hi</Message>
-            <AnotherSetting>setting</AnotherSetting>
-        </Error>
+    <Response>
```

Comment:
The first paragraph of this section also needs updating. Would you be able to make that change too?

This no longer lines up with the example:

>Error responses in the ``ec2Query`` protocol are wrapped within an XML root node named ``Errors``. A nested element, named ``Error``, contains the serialized error structure members.

Reply:
Will fix.